CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Contribute
+
+## How to translate
+
+1. Add your language to `[_config.yml](https://github.com/wp-cli/wp-cli.github.com/blob/master/_config.yml)`.
+2. Create a folder under the name of your appropriate language code (IETF language tag). e.g. ja or pt_BR
+3. Copy `index.md` into your language's directory.
+4. Translate `index.md`.
```

Comment:
Can we put this in the `README.md` for now? I don't think our `README.md` is so long that we need to consider separate files right now.